src/api/routes/auth/register.ts

```diff
@@ -225,6 +225,14 @@ router.post(
 		}
 
 		if (body.password) {
+                        if(body.password.length<8){
+                                throw FieldErrors({
+                                        password: {
+                                                code: "PASSWORD_REQUIREMENTS_MIN_LENGTH",
+                                                message: req.t("auth:register.PASSWORD_REQUIREMENTS_MIN_LENGTH")
```

Comment:
use a variable in the translation and insert `register.password.minLength` into it so it says something like `Must be at least <the config value> characters.`